.travis.yml

```diff
@@ -1,18 +1,10 @@
 ---
-# Verify this with: http://lint.travis-ci.org/
 language: ruby
 sudo: false
 # Delete dependency locks for matrix builds.
 before_install: rm Gemfile.lock || true
 script: bundle exec rake test
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.0
+  - 2.4.3
 env:
-  - PUPPET_VERSION="~> 3.8.0"
-  - PUPPET_VERSION="~> 4.6.0"
-matrix:
-  allow_failures:
-  - env: PUPPET_VERSION="~> 4.6.0"
-
+  - PUPPET_VERSION="~> 5.1.0"
```

Comment:
It looks like setting this variable has no effect if the Gemfile doesn't read it any more - let's drop it if so?